src/yuzu/main.cpp

```diff
@@ -269,6 +270,15 @@ bool GMainWindow::CheckDarkMode() {
 #endif // __linux__
 }
 
+static void font_menu_items(QList<QAction*> the_actions) {
+    foreach (QAction* action, the_actions) {
+        if (action->menu()) {
```

Comment:
Maybe assign this to a local variable, so that `action->menu()` doesn't need to be called repeatedly like:


```suggestion
        if (auto* menu = action->menu()) {
```

You may also want to leave a comment indicating that this should be switched over to:

```suggestion
        if (auto* menu = QMenu::menuInAction(action)) {
```

when we move to Qt 6, since `menu()` is deprecated in it.